packages/client/src/views/MyGrants.vue

```diff
@@ -6,10 +6,10 @@
     <b-tab title="Assigned">
       <GrantsTable :showAssignedToAgency="selectedAgencyId"/>
     </b-tab>
-    <b-tab title="Rejected">
+    <b-tab title="Non-award">
```

Comment:
I believe this should actually be "Not Applying" since this tab only indicates grants that a user has decided not to apply to, in other words, they have Rejected them from the applying process, rather than the grant-award being rejected by the government.

@mhuang-usdr can you confirm which of these "Expected" versions is the correct one?

imo @LieAlbertTriAdrian these tabs should look like **Expected A** version.

## Current
<img width="400" alt="image" src="https://user-images.githubusercontent.com/6497366/218146628-5d2e7943-2d09-4931-b194-ea6f20faa281.png">

## Expected A
<img width="449" alt="image" src="https://user-images.githubusercontent.com/6497366/218146464-e1e1fba8-5030-468e-ab19-771c683d6480.png">


## Expected B
<img width="420" alt="image" src="https://user-images.githubusercontent.com/6497366/218146118-5b914c7e-b319-41ca-93f9-3fca9506b468.png">

Reply:
@as1729 Good point.

@mhuang-usdr Please let us know which one you expected to be and I'll apply the changes immediately :)

Reply:
@LieAlbertTriAdrian I expected A! Thanks for asking

Reply:
Thanks @mhuang-usdr for confirming.

Updated @as1729

Reply:
Thank you!